### PR TITLE
rm req for ords download; non-root for OL8

### PR DESCRIPTION
**Pull request by gotsysdba**

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]
please get review from @rhondaday  too

**Comment by gotsysdba** [comment]
@rhondaday, can you give this change a once-over and approve if good?

**Review by rhondaday — APPROVED** [review]
Approved.

**Pull request closed without merging**
